ale_python_interface/ale_c_wrapper.h

```diff
@@ -63,6 +63,12 @@ extern "C" {
     int size = ale->getRAM().size();
     memcpy(ram,ale_ram,size*sizeof(unsigned char));
   }
+  void setRAM(ALEInterface *ale,unsigned char *ram){
```

Comment:
This seems to satisfy a very specific use case, where you might want to set the whole RAM state but nothing else. I'm not sure what usefulness that has. Can you explain a little more what you have in mind? I would imagine setRAM(ale, memory_index, value) is a much more useful function.